include/checkpoint/cpt_env.h

```diff
@@ -13,6 +13,7 @@
 * See the Mulan PSL v2 for more details.
 ***************************************************************************************/
 
+#ifndef CONFIG_SHARE
```

Comment:
If we remove checkpoint c files in the Makefile, do we need to change these files? If we remove them in Makefile, they should not be compiled?

Reply:
It's true that I don't have to change the file definition in checkpoint anymore